api/test/integration/env/base/nginx-lb/Dockerfile

```diff
@@ -3,3 +3,5 @@ FROM nginx
 COPY nginx.conf /etc/nginx/nginx.conf
 
 ADD entrypoint.sh /scripts/entrypoint.sh
+
+HEALTHCHECK --interval=30s --timeout=3s CMD curl --fail http://localhost/ || exit 1
```

Comment:
The interval seems a little bit high to me, if nginx fails at the start because the server isn't up yet, it will wait 30s to try again. We could reduce that number to 5/10 to avoid waiting too much.

The manager and agent base images healthchecks use an interval of `1s` and a timeout of `30s`, we could use similar values. Moreover, they specify a number of retries (900), we may want to set one to avoid unlimited tries. 

Lastly, I'm not sure if there is a policy for this, but perhaps we should add the EOF line

Reply:
I agree with the suggestion to reduce the interval value, a 1-second interval seems too often, but it shouldn't affect the environment.

The timeout is ok because nginx solves this call in localhost, it just returns an HTML standard page, so the response is immediate.

I don't know if adding EOF line obeys a standard. We can delete it to maintain the format of the prior version.